### DISCUSSION TITLE
Public Invoice

### DISCUSSION THREAD

**Discussion by balt5r** [btcpayserver/btcpayserver#2214] [General]
Would be nice to have a public link per invoice, for the user accounting.
With my company information that I could add on the dashboard.

**Comment by NicolasDorier** [comment]
ping @dennisreimann @dstrukt I think showing something with a design quite similar to what has been done for payment request would be nice.

I think only consumer facing info are needed typically: Addresses paid, order Id, rate.

Shall we allow the merchants to customize this as we do for payment requests?

**Reply by balt5r** [reply]
Merchant info are important as well (Legal name, Legal Address, email...), to make the invoice "valid"

**Reply by dennisreimann** [reply]
Yes. And it would be good to provide this info on the store level so that it doesn't have to be entered for each invoice.

**Reply by kenerik** [reply]
Here in Norway, and many European countries.
There is a legal requirement to give a valid receipt:
Date, VAT number, Purchased Item description, Company name, full Company address, amount in native currency, Unique receipt number and payment method.
Create a custom CSS template as @NicolasDorier  suggest? @dennisreimann

**Comment by Kukks** [comment]
I am pretty sure this was possible in the past but you removed it at some point @NicolasDorier?

**Comment by NicolasDorier** [comment]
Yes I removed it because there was some confidential info there. (say people putting secret things in redirect urls)

**Comment by dstrukt** [comment]
Just following up on this.

Is this something we want to proceed with?

**Reply by dstrukt** [reply]
@NicolasDorier @Kukks ^

Security/privacy concerns aside, while important, so I wanted to confirm with both of you.

I like the concept, and we could easily add any additions to both admin/user facing views.
